### PR TITLE
[DCOS-37832] Update Elastic to 5.6.9

### PR DESCRIPTION
**Pull request by mpereira**

### CODE REVIEW

File: frameworks/elastic/universe/resource.json [hunk 1]
```diff
@@ -5,7 +5,7 @@
       "libmesos-bundle-tar-gz": "{{libmesos-bundle-url}}",
       "bootstrap-zip": "{{artifact-dir}}/bootstrap.zip",
       "scheduler-zip": "{{artifact-dir}}/elastic-scheduler.zip",
-      "statsd-plugin-zip": "https://github.com/mesosphere/elasticsearch-statsd-plugin/releases/download/{{elastic-statsd-version}}/elasticsearch-statsd-{{elastic-statsd-version}}.zip",
+      "statsd-plugin-zip": "https://github.com/mesosphere/elasticsearch-statsd-plugin/releases/download/{{elastic-statsd-version}}/elasticsearch-statsd-{{elastic-statsd-version}}.zip",
```
[the remark this answers is not in the document]
Comment (mpereira) [reply frameworks/elastic/universe/resource.json]: Yes, I removed it here https://github.com/mesosphere/dcos-commons/pull/2536/commits/f0e0ecdc8666f57962f7eebef0e081129c8ceb75

File: frameworks/elastic/src/test/java/com/mesosphere/sdk/elastic/scheduler/ServiceTest.java [hunk 2]
```diff
@@ -76,8 +76,8 @@ private ServiceTestRunner getDefaultRunner() {
                         "some.thing=true",
                         "ZONE", "us-east-1a")
                 .setBuildTemplateParams(
-                        "elastic-version", "5.6.5",
-                        "elastic-statsd-version", "5.6.5.0", // NOPMD
-                        "support-diagnostics-version", "6.2");
+                        "elastic-version", "5.6.9",
```
[the remark this answers is not in the document]
Comment (mpereira) [reply frameworks/elastic/src/test/java/com/mesosphere/sdk/elastic/scheduler/ServiceTest.java]: Hm, I'd rather having realistic values here, but don't have a strong preference https://github.com/mesosphere/dcos-commons/pull/2536/commits/7fd62af7458d8e08216ae62d692e6dad3d5e3d51

[the remark this answers is not in the document]
Comment (elezar) [reply frameworks/elastic/src/test/java/com/mesosphere/sdk/elastic/scheduler/ServiceTest.java]: I'm ok with realistic values, but they don't have to be the same as what is define in `versions.sh`. Are you OK with the `1.2.3` and `1.2.3.0` strings?

[the remark this answers is not in the document]
Comment (mpereira) [reply frameworks/elastic/src/test/java/com/mesosphere/sdk/elastic/scheduler/ServiceTest.java]: Yes, that's what I used in the commit I pushed https://github.com/mesosphere/dcos-commons/commit/7fd62af7458d8e08216ae62d692e6dad3d5e3d51

**Comment by mpereira** [comment]
Thank you for taking the time to review it!

**Pull request merged**